man/journal-upload.conf.xml

```diff
@@ -60,6 +60,70 @@
         <xi:include href="version-info.xml" xpointer="v232"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>BatchMaxBytes=</varname></term>
+
+        <listitem><para>Max payload size of journal entries batch in a single HTTP request.</para>
+
+        <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>BatchMaxEntries=</varname></term>
+
+        <listitem><para>Max number of journal entries in a batch in a single HTTP request.</para>
+
+        <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>BatchTimeoutSec=</varname></term>
+
+        <listitem><para>Max time in seconds to wait since previous request before sending data.</para>
+
+        <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>Compression=</varname></term>
+
+        <listitem><para>Compression algorithm to apply to logs data before sending.
+        Supported values are <literal>ZSTD</literal>, <literal>XZ</literal>, <literal>LZ4</literal> and <literal>NONE</literal>.
+        Default is <literal>NONE</literal></para>
+
+        <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>CompressionLevel=</varname></term>
+
+        <listitem><para>Compression level is an integer, which defines a level of journal entries compression.
+        Each algorithm has it's own range of allowed values:  XZ - [1 - 9], ZSTD - [1 - 22], LZ4 - [1 - 9].
+        Default value: -1, which set's a default compression level for each compresion algorithm.</para>
```

Comment:
`Defaults to -1, and the default compression level for each compression algorithm will be used.`

Please also mention that this will be ignored when `CompressionLevel=none`.